src/situations/accueil/infra/depot_ressources_accueil.js

```diff
@@ -21,13 +28,27 @@ const batiments = {
   tri
 };
 
+const competences = {
+  comprehension_consigne: comprehensionConsigne,
+  rapidite: rapidite,
+  comparaison_tri: comparaisonTri,
+  vigilance_controle: vigilanceControle,
+  organisation_methode: organisationMethode,
+  attention_concentration: attentionConcentration,
+  perseverance: perseverance
+};
+
 export default class DepotRessourcesAccueil extends DepotRessources {
   constructor (chargeurs) {
     super(chargeurs);
-    this.charge([fondAccueil, personnage, precedent, suivant, punaise, casque]);
+    this.charge([fondAccueil, personnage, precedent, suivant, punaise, casque, comprehensionConsigne, rapidite, comparaisonTri, vigilanceControle, organisationMethode, attentionConcentration, perseverance]);
     this.charge(Object.values(batiments));
   }
 
+  pictoCompetences (competenceForte) {
+    return competences[competenceForte];
```

Comment:
Il manque l'appel à la fonction ressource non ?

`return this.ressource(competences[competenceForte]);`

Reply:
Hmm tu as testé ? Je comprend pas pourquoi il faudrait faire appel à la fonction, je retourne le path et dans `fin.vue` je `return this.depotRessources.pictoCompetences(competencesFortes);` , d'ailleurs en testant rapidement en ajoutant `return this.ressource(competences[competenceForte]);` il retourne `src="[object HTMLImageElement]"` dans mon dom.